docs/chapters.html

```diff
@@ -30,7 +30,6 @@
       const playerVTT = cloudinary.videoPlayer('player-vtt', {
         cloud_name: 'demo',
         publicId: 'snow_horses',
-        info: { title: 'Snow Horses', subtitle: 'A movie about horses' },
```

Comment:
I think its a good idea to keep one of them, to see how chapters would work with title and sub title.